Bottom: 4959edb430c3971d4844463aa0a9889364d5a09f
Top:    4569c22716e055ac0757021c4d22b0cdc8cb878b
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-04-30 14:43:27 -0500

DO NOT SUBMIT: Temporary changes for development

---
diff --git a/docker/buildbot-worker-ntel/Dockerfile b/docker/buildbot-worker-ntel/Dockerfile
index c85bf86..0af7dd9 100644
--- a/docker/buildbot-worker-ntel/Dockerfile
+++ b/docker/buildbot-worker-ntel/Dockerfile
@@ -83,6 +83,13 @@ RUN useradd --comment "Buildbot Server" --home-dir "/home/buildbot" --shell "/bi
 &&  chown -v -R buildbot:buildbot "/buildbot" \
 &&  chown -v -R buildbot:buildbot "/home/buildbot"
 
+RUN DEBIAN_FRONTEND=noninteractive \
+    apt-get update \
+&&  apt-get -y install -q \
+       vim \
+       strace \
+       tmux
+
 USER buildbot
 
 CMD ["/home/buildbot/start.sh"]
diff --git a/docker/buildbot-worker-ntel/buildbot/.bashrc b/docker/buildbot-worker-ntel/buildbot/.bashrc
new file mode 100644
index 0000000..c0da9fd
--- /dev/null
+++ b/docker/buildbot-worker-ntel/buildbot/.bashrc
@@ -0,0 +1 @@
+export TERM=xterm-256color
diff --git a/docker/buildbot-worker-ntel/buildbot/.tmux.conf b/docker/buildbot-worker-ntel/buildbot/.tmux.conf
new file mode 100644
index 0000000..01b0b36
--- /dev/null
+++ b/docker/buildbot-worker-ntel/buildbot/.tmux.conf
@@ -0,0 +1,37 @@
+# Prefix
+unbind C-b
+set -g prefix `
+bind-key ` send-prefix
+
+# 0 is too far from ` ;)
+set -g base-index 1
+
+# Automatically set window title
+set-window-option -g automatic-rename on
+set-option -g set-titles on
+set -g renumber-windows on
+
+set -g default-terminal screen-256color
+set -g status-keys vi
+set -g history-limit 10000
+
+setw -g mode-keys vi
+#setw -g mode-mouse on
+setw -g monitor-activity on
+
+# No delay for escape key press
+set -sg escape-time 0
+
+# Reload tmux config
+bind r source-file ~/.tmux.conf
+
+# THEME
+set -g status-bg black
+set -g status-fg white
+set -g window-status-current-bg white
+set -g window-status-current-fg black
+set -g window-status-current-attr bold
+set -g status-interval 60
+set -g status-left-length 30
+set -g status-left '#[fg=green](#S) #(whoami)'
+set -g status-right '#[fg=yellow]#(cut -d " " -f 1-3 /proc/loadavg)#[default] #[fg=white]%H:%M#[default]'
diff --git a/docker/buildbot-worker-ntel/buildbot/.vimrc b/docker/buildbot-worker-ntel/buildbot/.vimrc
new file mode 100644
index 0000000..d2b7bc6
--- /dev/null
+++ b/docker/buildbot-worker-ntel/buildbot/.vimrc
@@ -0,0 +1,108 @@
+"compatible     behave very Vi compatible (not advisable)
+"       set nocp        cp
+set nocp
+
+"wrapscan       search commands wrap around the end of the buffer
+"       set ws  nows
+set wrapscan
+
+"incsearch      show match for partly typed search command
+"       set nois        is
+set incsearch
+
+"wrap   long lines wrap
+"       set wrap        nowrap
+set nowrap
+
+"number show the line number for each line
+"       (local to window)
+"       set nonu        nu
+set nu
+
+"background     "dark" or "light"; the background color brightness
+"       set bg=light
+set bg=dark
+
+"filetype       type of file; triggers the FileType event when set
+"       (local to buffer)
+"       set ft=vim
+filetype plugin indent on
+
+"syntax name of syntax highlighting used
+"       (local to buffer)
+"       set syn=vim
+syntax on
+
+"hlsearch       highlight all matches for the last used search pattern
+"       set nohls       hls
+set hls
+
+"cursorline     highlight the screen line of the cursor
+"       (local to window)
+"       set nocul       cul
+set cursorline
+
+"showcmd        show (partial) command keys in the status line
+"       set nosc        sc
+set showcmd
+
+"showmode       display the current mode in the status line
+"       set smd nosmd
+set showmode
+
+"ruler  show cursor position below each window
+"       set ru  noru
+set ruler
+
+"backspace      specifies what <BS>, CTRL-W, etc. can do in Insert mode
+"       set bs=indent,eol,start
+set backspace=indent,eol,start
+
+"tabstop        number of spaces a <Tab> in the text stands for
+"       (local to buffer)
+"       set ts=8
+set ts=8
+
+"shiftwidth     number of spaces used for each step of (auto)indent
+"       (local to buffer)
+"       set sw=8
+set sw=4
+
+"smarttab       a <Tab> in an indent inserts 'shiftwidth' spaces
+"       set nosta       sta
+set smarttab
+
+"softtabstop    if non-zero, number of spaces to insert for a <Tab>
+"       (local to buffer)
+"       set sts=0
+set softtabstop=-1
+
+"shiftround     round to 'shiftwidth' for "<<" and ">>"
+"       set nosr        sr
+set shiftround
+
+"expandtab      expand <Tab> to spaces in Insert mode
+"       (local to buffer)
+"       set noet        et
+set expandtab
+
+"autoindent     automatically set the indent of a new line
+"       (local to buffer)
+"       set noai        ai
+set autoindent
+
+"smartindent    do clever autoindenting
+"       (local to buffer)
+"       set nosi        si
+set smartindent
+
+"fileformats    list of file formats to look for when editing a file
+"       set ffs=unix,dos
+set fileformats=unix,dos
+
+"backup keep a backup after overwriting a file
+"       set nobk        bk
+set nobackup
+
+"Key Maps
+map! <S-Insert> <C-r>+
